net/fail2ban/Makefile

```diff
@@ -0,0 +1,115 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fail2ban
+PKG_VERSION:=0.11.2
+PKG_RELEASE:=2
+PKGARCH:=all
+
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
```

Comment:
?

Take a look at e.g. [this Makefile (netdata)](https://github.com/openwrt/packages/blob/master/admin/netdata/Makefile#L19)